crm_lead_currency/views/crm_lead_views.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" ?>
+<odoo>
+
+  <record id="crm_lead_view_form" model="ir.ui.view">
+    <field name="name">crm.lead.form</field>
+    <field name="model">crm.lead</field>
+    <field name="inherit_id" ref="crm.crm_lead_view_form" />
+    <field name="arch" type="xml">
+      <field name="partner_id" position="before">
+        <field name="customer_currency_id" />
+        <field name="is_same_currency" invisible="1" />
+        <field
+                    name="amount_customer_currency"
+                    attrs="{'invisible': [('is_same_currency', '=', True)]}"
+                />
+      </field>
+      <field name="expected_revenue" position="attributes">
+        <attribute
+                    name="attrs"
+                >{'readonly': [('is_same_currency', '=', False)]}</attribute>
+        <attribute name="force_save">1</attribute>
+      </field>
+    </field>
+  </record>
+
+  <!-- In kanban display the amount in the customer currency as suitable -->
+  <record id="crm_case_kanban_view_leads" model="ir.ui.view">
+    <field name="name">crm.lead.kanban.lead</field>
+    <field name="model">crm.lead</field>
+    <field name="inherit_id" ref="crm.crm_case_kanban_view_leads" />
+    <field name="priority">99</field>
+    <field name="arch" type="xml">
+
+      <field name="expected_revenue" position="after">
+        <field name="is_same_currency" />
+        <field name="amount_customer_currency" />
+        <field name="customer_currency_id" />
+      </field>
+
+      <xpath expr="//templates//field[@name='expected_revenue']" position="replace">
+        <t t-if="record.is_same_currency.raw_value">
+          <field
+                        name="expected_revenue"
```

Comment:
Please fix indentation. Since this version now uses pre-commit, correct indentation is important, to avoid mismatches like this one between nodes indentation and attributes indentation.

This could be on a separated commit before precommit.

Reply:
> This could be on a separated commit before precommit.

This was not done. Formatting on a separated commit is important, because if not done, it would much harder to determine which where the actual changes.